docker/Dockerfile-flower

```diff
@@ -0,0 +1,18 @@
+FROM registry.fedoraproject.org/fedora:33
+LABEL maintainer="Factory 2.0"
+
+WORKDIR /src
+RUN dnf -y install \
+    --setopt=deltarpm=0 \
+    --setopt=install_weak_deps=false \
+    --setopt=tsflags=nodocs \
+    redis \
+    python3-pip
+RUN pip3 install flower==0.9.7
+
+COPY . .
+EXPOSE 7777 5555
+CMD ["cd cachito/workers/tasks ; \
+    /usr/bin/redis-server --port=7777 & \
```

Comment:
Why do we need redis here?

We are using rabbitmq as our broker. Shouldn't flower point to our celery broker?

Moreover, if we were to run a second server, it would be nice to have it running in a dedicated container.

Reply:
Changed to point to celery broker